### PR TITLE
V1.1.0 wip

### PR DESCRIPTION
**Pull request by tetron**

### CODE REVIEW

**Comment by mr-c** [comment]
Woohoo!

**Comment by tetron** [comment]
Uses features from schema salad master, so we'll need to cut a release over there.

Conformance tests being added to 1.0 will need to be copied over to 1.1.

Lots of 1.1 schema fixes still pending, some of which require a corresponding implementation in cwltool.

File: v1.1.0-dev1/CommandLineTool.yml [hunk 1]
```diff
@@ -36,60 +36,19 @@ $graph:
...
-        * 13 July 2016: Mark `baseCommand` as optional and update descriptive
-          text.
-        * 14 November 2016: Clarify [SoftwareRequirement](#SoftwareRequirement)
-          `spec` fields.
+      ## Introduction to CWL Command Line Tool standard v1.1.0-dev1
+
+      This specification represents the latest stable release from the
+      CWL group.  Since the v1.0 release, v1.1 introduces the
+      following updates to the CWL Command Line Tool standard.
+      Documents should to use `cwlVersion: v1.1` to make use of new
```
Comment (mr-c) [inline v1.1.0-dev1/CommandLineTool.yml]: `cwlVersion: v1.1.0-dev1`

File: v1.1.0-dev1/CommandLineTool.yml [hunk 2]
```diff
@@ -207,31 +167,39 @@ $graph:
...
       type: boolean?
+      default: true
       doc: |
         If `ShellCommandRequirement` is in the requirements for the current command,
         this controls whether the value is quoted on the command line (default is true).
         Use `shellQuote: false` to inject metacharacters for operations such as pipes.
 
+        If `shellQuote` is true or null, the implementation must not
+        permit interpretation of any shell metacharacters or directives.
+
```
Comment (mr-c) [inline v1.1.0-dev1/CommandLineTool.yml]: Changelog entry please

File: v1.1.0-dev1/CommandLineTool.yml [hunk 3]
```diff
@@ -867,7 +837,7 @@ $graph:
     - name: listing
       type:
         - type: array
-          items: [File, Directory, Dirent, string, Expression]
+          items: ["null", File, Directory, Dirent, string, Expression]
```
Comment (mr-c) [inline v1.1.0-dev1/CommandLineTool.yml]: Changelog entry please

File: v1.1.0-dev1/CommandLineTool.yml [hunk 4]
```diff
@@ -877,7 +847,7 @@ $graph:
         designated output directory prior to executing the command line tool.
 
         May be an expression.  If so, the expression return value must validate
-        as `{type: array, items: [File, Directory]}`.
+        as `{type: array, items: ["null", File, Directory, Dirent]}`.
```
Comment (mr-c) [inline v1.1.0-dev1/CommandLineTool.yml]: Changelog entry please

File: v1.1.0-dev1/Process.yml [hunk 5]
```diff
@@ -423,20 +553,42 @@ $graph:
 
 
 - name: InputSchema
-  extends: SchemaBase
+  extends: [Labeled, BindsInput]
   type: record
   abstract: true
+  fields:
+    - name: name
```
Comment (mr-c) [inline v1.1.0-dev1/Process.yml]: Changelog entry please

**Pull request merged**